test/metabase/query_processor/middleware/annotate_test.clj

```diff
@@ -460,15 +460,14 @@
   (qp.store/with-metadata-provider meta/metadata-provider
     (testing (str "if a driver is kind enough to supply us with some information about the `:cols` that come back, we "
                   "should include that information in the results. Their information should be preferred over ours")
-      (is (=? {:cols [{:name           "totalEvents"
-                       :display_name   "Total Events"
+      (is (=? {:cols [{:display_name   "Total Events"
                        :base_type      :type/Text
                        :effective_type :type/Text
                        :source         :aggregation
                        :field_ref      [:aggregation 0]}]}
               (add-column-info
                (lib.tu.macros/mbql-query venues {:aggregation [[:metric 1]]})
-               {:cols [{:name "totalEvents", :display_name "Total Events", :base_type :type/Text}]}))))))
```

Comment:
Prior to this PR the name of metric was slapped onto metrics metadata in `lib.metadata.calculation/metadata-method :metric` implementation (called in the trace of `add-column-info`). Because (1) original aggregation's name is used now and (2) `[:metric 1]` is not defined (ie. no original aggregation) and (3) `:name` of `ours` (ie. coming from query) as per `metabase.query-processor.middleware.annotate/merge-col-metadata` is prefered it does not make sense to test for that here anymore.